packages/appeals-service-api/src/db/seed/lpa-appeal-case-data-dev.js

```diff
@@ -0,0 +1,496 @@
+const { pickRandom, datesNMonthsAgo, datesNMonthsAhead } = require('./util');
+
+const appealSubmissionDraft = {
+	// ID in Cosmos, see dev/data
+	id: '89aa8504-773c-42be-bb68-029716ad9756'
+};
+
+const lpaAppealIds = {
+	appealOne: '756d6bfb-dde8-4532-a041-86c226a23b01',
+	appealTwo: '756d6bfb-dde8-4532-a041-86c226a23b02',
+	appealThree: '756d6bfb-dde8-4532-a041-86c226a23b03',
+	appealFour: '756d6bfb-dde8-4532-a041-86c226a23b04',
+	appealFive: '756d6bfb-dde8-4532-a041-86c226a23b05',
+	appealSix: '756d6bfb-dde8-4532-a041-86c226a23b06',
+	appealSeven: '756d6bfb-dde8-4532-a041-86c226a23b07',
+	appealEight: '756d6bfb-dde8-4532-a041-86c226a23b08',
+	appealNine: '756d6bfb-dde8-4532-a041-86c226a23b09',
+	appealTen: '756d6bfb-dde8-4532-a041-86c226a23b10',
+	appealEleven: '756d6bfb-dde8-4532-a041-86c226a23b11',
+	appealTwelve: '756d6bfb-dde8-4532-a041-86c226a23b12',
+	appealThirteen: '756d6bfb-dde8-4532-a041-86c226a23b13',
+	appealFourteen: '756d6bfb-dde8-4532-a041-86c226a23b14',
+	appealFifteen: '756d6bfb-dde8-4532-a041-86c226a23b15'
+};
+
+/**
+ * @type {import('@prisma/client').Prisma.AppealCreateInput[]}
+ */
+const lpaAppeals = [
+	{ id: lpaAppealIds.appealOne },
+	{ id: lpaAppealIds.appealTwo },
+	{ id: lpaAppealIds.appealThree },
+	{ id: lpaAppealIds.appealFour },
+	{ id: lpaAppealIds.appealFive },
+	{ id: lpaAppealIds.appealSix },
+	{ id: lpaAppealIds.appealSeven },
+	{ id: lpaAppealIds.appealEight },
+	{ id: lpaAppealIds.appealNine },
+	{ id: lpaAppealIds.appealTen },
+	{ id: lpaAppealIds.appealEleven },
+	{ id: lpaAppealIds.appealTwelve },
+	{ id: lpaAppealIds.appealThirteen },
+	{ id: lpaAppealIds.appealFourteen },
+	{ id: lpaAppealIds.appealFifteen },
+	{
+		id: appealSubmissionDraft.id,
+		legacyAppealSubmissionId: appealSubmissionDraft.id,
+		legacyAppealSubmissionState: 'DRAFT'
+	}
+];
+
+/**
+ * @type {import('@prisma/client').Prisma.AppealCaseCreateInput[]}
+ */
+const lpaAppealCaseData = [
+	{
+		Appeal: {
+			connect: { id: lpaAppealIds.appealOne }
+		},
+		caseReference: '0000000',
+		LPACode: 'Q9999',
+		LPAName: 'System Test Borough Council',
+		appealTypeCode: 'HAS',
+		appealTypeName: 'Householder',
+		decision: 'refused',
+		originalCaseDecisionDate: pickRandom(datesNMonthsAgo(1)),
+		costsAppliedForIndicator: false,
+		LPAApplicationReference: '12/2323232/PLA',
+		siteAddressLine1: 'New appeal',
+		siteAddressLine2: null,
+		siteAddressTown: 'No due dates',
+		siteAddressCounty: 'Countyshire',
+		siteAddressPostcode: 'BS1 6PN',
+		questionnaireDueDate: null,
+		questionnaireReceived: null,
+		statementDueDate: null,
+		LPAStatementSubmitted: null,
+		finalCommentsDueDate: null,
+		LPACommentsSubmitted: null,
+		proofsOfEvidenceDueDate: null,
+		LPAProofsSubmitted: null,
+		outcome: null,
+		caseDecisionDate: null
```

Comment:
See `data-dev` (common appeal properties) - we should pull out common properties and spread them in (or create a function that returns an appeal based on a few properties).

Reply:
I've adopted the spread approach for now, a possible refactor for the future is using a builder